### PR TITLE
[9.x] Refactor validation closures  to arrow functions

### PR DESCRIPTION
**Pull request by Nothehi**
Refactor `Validation` closures to arrow functions

### CODE REVIEW

**Comment by henzeb** [comment]
Why?

**Comment by Nothehi** [comment]
From PHP 7.4, We can make closure functions with the fat arrow which is now more readable, shorter, and cleaner. I thought it would be a good idea to convert these closures to arrow functions.

File: src/Illuminate/Validation/Rules/DatabaseRule.php [hunk 1]
```diff
@@ -69,9 +69,13 @@ public function resolveTableName($table)
...
-            return implode('.', array_map(function (string $part) {
-                return trim($part, '.');
-            }, array_filter([$model->getConnectionName(), $model->getTable()])));
+            return implode(
+                '.',
+                array_map(
+                    fn (string $part) => trim($part, '.'),
+                    array_filter([$model->getConnectionName(), $model->getTable()])
+                )
+            );
```
Comment (driesvints) [inline src/Illuminate/Validation/Rules/DatabaseRule.php:78]:
```suggestion
            return implode('.', array_map(
                fn (string $part) => trim($part, '.'),
                array_filter([$model->getConnectionName(), $model->getTable()])
            ));
```

**Review by driesvints — CHANGES_REQUESTED** [review]

**Comment by taylorotwell** [comment]
It's OK.

**Pull request closed without merging**